### ISSUE TITLE
Camera on uncrouch get too adjusted to capsule end

### ISSUE THREAD

**Issue by Hevedy** [ProjectBorealis/PBCharacterMovement#18]
When uncrouch the camera get too close to the end of the capsule, as result on collisions the camera ends inside meshes or on middle point.

A parameter to limit the distance between capsule half size and % between end could be a good point

**Comment by mastercoms** [comment]
This is in our internal code, but hasn't been pushed yet. We're currently heads down working on content, so we can't commit time to updating the open source version at this time.

**Comment by azradric** [comment]
```
void APBPlayerCharacter::RecalculateBaseEyeHeight()
{
	BaseEyeHeight = FMath::Lerp(GetDefault<APawn>(GetClass())->BaseEyeHeight, CrouchedEyeHeight, MovementPtr->CurrentCroachAlpha);
}
```

In the `DoCrouchResize`:
```
CurrentCroachAlpha += TargetAlphaDiff;
```

In the `DoUnCrouchResize`:
```
CurrentCroachAlpha -= TargetAlphaDiff;
```

Solved.

**Comment by mastercoms** [comment]
Fixed in 74791cb2b465c5b1e0b3bd9beb91ac1d6accf570
